src/code42cli/cmds/search/extraction.py

```diff
@@ -100,10 +100,7 @@ def _format_output():
         if total_events > 10 or force_pager:
             click.echo_via_pager(_format_output())
         else:
-            for page in _format_output():
-                click.echo(page, nl=False)
-            if formatter.output_format == OutputFormat.TABLE:
-                click.echo()
+            formatter.echo_formatters_output(_format_output())
```

Comment:
I think you mean to use `echo_formatted_list()` here?

Reply:
agreed, this doesn't appear to be a defined function

Reply:
and we're now calling the format_output function twice on this data, which is causing it to escape the quotations and newlines